voxxed-microservices-2018/module-01.adoc

```diff
@@ -295,7 +295,7 @@ After "Add" is clicked, Grafana will test the connection to the data source.
 image::grafana_prometheus_data_source.png[grafana prometheus data source]
 
 From the top left menu, click on "Dashboards" and then "Import" to open the "Import Dashboard" window.
-Open a browser tab and navigate to `https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/0.7.0/metrics/examples/grafana/kafka-dashboard.json`.
+Open a browser tab and navigate to `https://raw.githubusercontent.com/strimzi/strimzi-kafka-operator/master/metrics/examples/grafana/strimzi-kafka.json`.
```

Comment:
Hum, hum, I'm still seeing no data in Grafana :( Also the drop-downs for selecting namespace etc. are empty :(

Reply:
Thanks a lot, I forgot to add one more step!